elements/pfe-clipboard/src/pfe-clipboard.scss

```diff
@@ -15,6 +15,7 @@ $LOCAL-VARIABLES: (
   icon--Width: pfe-var(icon-size),
   icon--Height: auto,
   icon--Margin: 0 0.4825rem 0 0, // 7.72px
+  icon--Color: #6a6e73
```

Comment:
Instead of hardcoding the hex, we should point this at a value in the map.  `pfe-var(ui-base)` maybe?  I posted a question in chat for Corey about it.

Reply:
Corey said `text--muted` 👍

Reply:
tied to text--muted!